generators/app/index.js

```diff
@@ -35,6 +36,7 @@ module.exports = class extends Generator {
 
   writing () {
     this._copy('templates/client/main.html')
+    this._copy('templates/client/main.scss')
```

Comment:
We do not want to set `scss` as default. Some CSS frameworks do not use `scss` and use `less` instead. Since this is the main task that first bootstraps the app, stylings stuff shouldn't be a default.

See `generators/styles/materialize` to see the template on installing a CSS framework for stylings.